etc/scripts/osikeys.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import licensedcode.index
+import json
+from licensedcode.cache import get_index
+from licensedcode.models import load_licenses
+from licensedcode.models import Rule
+from licensedcode.models import rules_data_dir
+
+def findOSIMap(license):
+    """
+    Return Scancode key mapped to OSI License `license`
+    """
+    idx = get_index()
+    osiKey = license['id']
+    matches = list(idx.match(query_string = osiKey))
+    if not matches:
+        return None
+    return matches[0].rule.license_expression, osiKey
```

Comment:
why also return osiKey? where does it come from ? Since you are calling with `license` you already know this value, you already know this, do you?